src/main/java/com/revature/controllers/CurriculumController.java

```diff
@@ -61,8 +61,8 @@
 		return ResponseEntity.ok(curriculumService.createCurriculum(curriculum));
 	}
 
-	@PutMapping("/{id}")
-	public ResponseEntity<Curriculum> updateCurriculum(@PathVariable("id") int id, @RequestBody Curriculum curriculum) {
+	@PutMapping
```

Comment:
Why was the id param removed here? It's convention to have it in the url

Reply:
we weren't actually using the id param but I can replace it